lib/task-data/tasks/install-debian.js

```diff
@@ -1,23 +1,24 @@
-// Copyright 2015, EMC, Inc.
+// Copyright 2017, Dell EMC, Inc.
 
 'use strict';
 
 module.exports = {
-    friendlyName: 'Install Ubuntu',
-    injectableName: 'Task.Os.Install.Ubuntu',
+    friendlyName: 'Install Debian/Ubuntu',
+    injectableName: 'Task.Os.Install.Debian',
     implementsTask: 'Task.Base.Os.Install',
-    optionsSchema: 'install-ubuntu.json',
+    optionsSchema: 'install-debian.json',
     options: {
         osType: 'linux', //readonly options, should avoid change it
-        profile: 'install-ubuntu.ipxe',
-        installScript: 'ubuntu-preseed',
+        osName: 'debian', //use "ubuntu" if you want to install ubuntu.
+        profile: 'install-debian.ipxe',
+        installScript: 'debian-preseed',
         installScriptUri: '{{ api.templates }}/{{ options.installScript }}?nodeId={{ task.nodeId }}', //jshint ignore: line
-        rackhdCallbackScript: 'ubuntu.rackhdcallback',
+        rackhdCallbackScript: 'debian.rackhdcallback',
         hostname: 'localhost',
         comport: 'ttyS0',
-        version: 'trusty',
-        repo: '{{file.server}}/ubuntu',
-        baseUrl: 'dists/{{ options.version }}/main/installer-amd64/current/images/netboot/ubuntu-installer/amd64',
+        version: 'stretch',
+        repo: '{{file.server}}/debian',
+        baseUrl: 'dists/{{ options.version }}/main/installer-amd64/current/images/netboot/debian-installer/amd64',
```

Comment:
We could change `repo` and `baseUrl` to 
```
repo: '{{file.server}}/{{options.osName}}',
baseUrl: 'dists/{{ options.version }}/main/installer-amd64/current/images/netboot/{{options.osName}}-installer/amd64',
```
So that the payload only needs `osName` and `version`.

Reply:
good i will change this part